ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<blockquote>(in the format <code>projects/[PROJECT_NAME]/locations/us-central1/models/[MODEL_ID</code>)</blockquote><p>missing "]" after MODEL_ID</p><p> </p><blockquote>regionalized hostname used for your model</blockquote><p>Can we say this is the regionalized hostname that you used when you trained the model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>No because the hostname includes the *-aiplatform.googleapis.com, whereas the region / location they specified earlier is part of the path.</p>